### PR TITLE
[release/1.3 backport] Make script work for both python2.7 and 3.6

### PR DESCRIPTION
**Pull request by dims**
Signed-off-by: Varsha Teratipally <teratipally@google.com>
(cherry picked from commit bf3dcfe3a8b94e854c9264491771485b70cbfd19)
Signed-off-by: Sebastiaan van Stijn <github@gone.nl>

### CODE REVIEW

**Comment by adisky** [comment]
LGTM!!!

**Review by estesp — APPROVED** [review]
LGTM

**Review by thaJeztah — APPROVED** [review]
LGTM

**Review by mikebrow — APPROVED** [review]
LGTM
pull issues are a reported known issue with us not being on 1.6 golang in this release will have to move up soon. 

Or not. Looks like Mr. @dims found a fix by disabling GO111MODULE support in prow.

File: cluster/gce/configure.sh [hunk 1]
```diff
@@ -19,6 +19,22 @@ set -o errexit
...
+if [[ "$(python -V 2>&1)" =~ "Python 2" ]]; then
+  # found python2, just use that
+  PYTHON="python"
+elif [[ -f "/usr/bin/python2.7" ]]; then
+  # System python not defaulted to python 2 but using 2.7 during migration
+  PYTHON="/usr/bin/python2.7"
+else
+  # No python2 either by default, let's see if we can find python3
+  PYTHON="python3"
+  if ! command -v ${PYTHON} >/dev/null 2>&1; then
```
Comment (fuweid) [inline cluster/gce/configure.sh:31]: using `command -v` is great!

**Review by fuweid — COMMENTED** [review]
LGTM

**Pull request merged**